### PR TITLE
CXF-8359: fixed mask for XML elements containing attributes

### PR DESCRIPTION
**Pull request by ashakirin**
Fixed mask for XML elements containing attributes

### CODE REVIEW

File: rt/features/logging/src/main/java/org/apache/cxf/ext/logging/MaskSensitiveHelper.java [hunk 1]
```diff
@@ -70,15 +71,19 @@ public String maskSensitiveElements(
             final Message message,
             final String originalLogString) {
         if (replacementsXML.isEmpty() && replacementsJSON.isEmpty()
-                || message == null
```
Comment (reta) [inline rt/features/logging/src/main/java/org/apache/cxf/ext/logging/MaskSensitiveHelper.java:73]: Why this condition was removed? It is exactly what will be returned on line #80 but with unnecessary use of `Optional::ofNullable(message)`?

Comment (ashakirin) [reply rt/features/logging/src/main/java/org/apache/cxf/ext/logging/MaskSensitiveHelper.java:73]: The statement "contentType = Optional.ofNullable(message).map(m -> (String) m.get(Message.CONTENT_TYPE))" handles null message out of the box. Null check on top is redundant

Comment (reta) [reply rt/features/logging/src/main/java/org/apache/cxf/ext/logging/MaskSensitiveHelper.java:73]: That is correct, so the question becomes: why do you need `Optional` at all? With the previous check in place, just:

```
final String contentType = (String) m.get(Message.CONTENT_TYPE));
if (contentType  == null) {
      return originalLogString;
}
```

It looks as straightforward as it gets (no need to create unnecessary instances).

Comment (ashakirin) [reply rt/features/logging/src/main/java/org/apache/cxf/ext/logging/MaskSensitiveHelper.java:73]: Matter of taste, for me Optional alternative is a bit compacter

**Pull request merged**